cmd/warrant/main.go

```diff
@@ -165,6 +166,21 @@ func (env *ServiceEnv) InitEventDB(config config.Config) error {
 		return nil
 	}
 
+	if config.Eventstore.Tigris.URL != "" {
```

Comment:
Since `URL` is not a required field, can we check for another field here? Maybe `Project`?

Reply:
Fixed.